templates/zerver/help/starring-messages.md

```diff
@@ -0,0 +1,24 @@
+# Starring a message
+**Starring messages is a good way to keep track of important messages, such as tasks you need to go back to,
+ or documents you reference often.**
+### Star a message
+You can star a message on desktop clients by hovering over any message. The timestamp of the message should be replaced with three context buttons while hovering, one of which is a star. Click on this to star the message.
```

Comment:
I am not sure we should limit to just "desktop clients" here. This feature applies to all clients I think.